installation_stevelaskaridis.sh

```diff
@@ -75,82 +72,122 @@ function install_new_software()
     add_new_ppas
     # Tools
     CURRENT='unity-tweak-tool'
-    sudo apt-get install -y $CURRENT
-    EXIT_CODE=$?
-    write_log $CURRENT $EXIT_CODE 
+    if ! APPLOCATION="$(type -p "$CURRENT")" || [ -z "$APPLOCATION" ]; then
```

Comment:
I used type instead of which for the reasons that are named in this post: http://stackoverflow.com/questions/592620/check-if-a-program-exists-from-a-bash-script . I can change it if you want.